### PR TITLE
Indicate that abs() of the minimum integer value for signed dtypes is undefined

### PR DESCRIPTION
**Pull request by asmeurer**

### CODE REVIEW

File: spec/API_specification/elementwise_functions.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ A conforming implementation of the array API standard must provide and support t
 (function-abs)=
 ### abs(x, /)
 
-Calculates the absolute value for each element `x_i` of the input array `x` (i.e., the element-wise result has the same magnitude as the respective element in `x` but has positive sign).
+Calculates the absolute value for each element `x_i` of the input array `x` (i.e., the element-wise result has the same magnitude as the respective element in `x` but has positive sign). For signed integer data types, the `abs` of smallest representable integer is implementation-defined.
```
Comment (kgryte) [inline spec/API_specification/elementwise_functions.md:25]: Should we move this to the special cases section? Or maybe just move to a note?

**Comment by asmeurer** [comment]
Should we add the same note for `negative`? 

I noticed that NumPy uses `-(min value) = min value` (e.g., `np.negative(-128, dtype=np.int8) == -128`) and `abs(min value) = min value`. This has the properties that `-x + x` always equals `0`, and `abs(x) = -x` for `x < 0`.

**Comment by rgommers** [comment]
> Should we add the same note for `negative`?

Seems reasonable to me.

File: spec/API_specification/elementwise_functions.md [hunk 2]
```diff
@@ -24,6 +24,11 @@ A conforming implementation of the array API standard must provide and support t
 
 Calculates the absolute value for each element `x_i` of the input array `x` (i.e., the element-wise result has the same magnitude as the respective element in `x` but has positive sign).
 
+```{note}
+
+For signed integer data types, the `abs` of smallest representable integer is implementation-dependent.
```
Comment (kgryte) [inline spec/API_specification/elementwise_functions.md:29]:
```suggestion
For signed integer data types, the absolute value of smallest representable integer is implementation-dependent.
```

File: spec/API_specification/elementwise_functions.md [hunk 3]
```diff
@@ -24,6 +24,11 @@ A conforming implementation of the array API standard must provide and support t
 
 Calculates the absolute value for each element `x_i` of the input array `x` (i.e., the element-wise result has the same magnitude as the respective element in `x` but has positive sign).
 
+```{note}
+
```
Comment (kgryte) [inline spec/API_specification/elementwise_functions.md:28]:
```suggestion
```

File: spec/API_specification/elementwise_functions.md [hunk 4]
```diff
@@ -24,6 +24,11 @@ A conforming implementation of the array API standard must provide and support t
...
+For signed integer data types, the absolute value of smallest representable integer is implementation-dependent.
```
Comment (kgryte) [inline spec/API_specification/elementwise_functions.md:28]:
```suggestion
For signed integer data types, the absolute value of minimum representable integer is implementation-dependent.
```

File: spec/API_specification/elementwise_functions.md [hunk 5]
```diff
@@ -24,6 +24,10 @@ A conforming implementation of the array API standard must provide and support t
 
 Calculates the absolute value for each element `x_i` of the input array `x` (i.e., the element-wise result has the same magnitude as the respective element in `x` but has positive sign).
 
+```{note}
+For signed integer data types, the absolute value of minimum representable integer is implementation-dependent.
```
Comment (kgryte) [inline spec/API_specification/elementwise_functions.md:28]:
```suggestion
For signed integer data types, the absolute value of the minimum representable integer is implementation-dependent.
```

**Review by kgryte — APPROVED** [review]
LGTM.

**Comment by kgryte** [comment]
I'll make a separate PR for `negative`.

**Pull request merged**